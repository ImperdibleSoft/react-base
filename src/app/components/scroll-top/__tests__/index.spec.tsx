import * as React from 'react';
import { shallow } from 'enzyme';

import ScrollTop from '..';

describe('ScrollTop', () => {
  it('should render its children', () => {
    const component = shallow(
      <ScrollTop>
        <p>Test</p>
      </ScrollTop>
    );
    expect(component).toMatchSnapshot();
  });
});
