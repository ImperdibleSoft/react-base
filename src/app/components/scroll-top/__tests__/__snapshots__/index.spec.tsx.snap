// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollTop should render its children 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
